packages/calypso-polyfills/package.json

```diff
@@ -25,9 +25,7 @@
 	},
 	"dependencies": {
 		"core-js": "^3.6.3",
-		"element-closest": "^3.0.2",
 		"isomorphic-fetch": "^2.2.1",
-		"regenerator-runtime": "^0.13.3",
-		"svg4everybody": "^2.1.9"
+		"regenerator-runtime": "^0.13.3"
```

Comment:
I'm wondering if we could remove `regenerator-runtime` from the Node.js polyfill. Node.js has had native generator and async/await support for a long time, so this should not be needed.

Or are the 3rd-party NPM packages that access the `regeneratorRuntime` global? The trouble with the `regenerator-runtime` package is that it creates a global variable in addition to exporting stuff.

Reply:
Good point, I'll check.

Reply:
Looks like we can remove it. The webpacked version of the polyfill file is:

```
/***/ "./packages/calypso-polyfills/node.js":
/*!********************************************!*\
  !*** ./packages/calypso-polyfills/node.js ***!
  \********************************************/
/***/ ((__unused_webpack_module, __unused_webpack_exports, __webpack_require__) => {

// Polyfills required for the server build of Calypso.
__webpack_require__(/*! isomorphic-fetch */ "isomorphic-fetch");

/***/ }),
```

Webpack removes both `core-js/stable` and `regenerator-runtime/runtime`.

I suggest we leave `core-js/stable` in case a future new version of Node (o core-js) adds some polyfill with bugfixes or something like that, but drop `regenerator-runtime/runtime`.

Reply:
> Webpack removes both `core-js/stable`

The `core-js/stable` import should be transformed by the `@babel/plugin-transform-runtime` into a series of imports of individual polyfills that are really needed. Maybe, in our case, the set of needed polyfills turns out the be empty?

That can change once new features are added to the standard and our version of Node.js doesn't support them. That's why the `core-js/stable` import should always be there.